src/CoreWCF.MSMQ/src/CoreWCF/Channels/MsmqBindingBase.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+
+namespace CoreWCF.Channels
+{ 
+    public abstract class MsmqBindingBase : Binding
+    {
+        internal MsmqBindingElementBase _transport;
+
+        protected MsmqBindingBase()
+        { }
+
+        // [DefaultValue(typeof(System.TimeSpan), MsmqDefaults.ValidityDurationString)]
+        //public TimeSpan ValidityDuration
+        //{
+        //    get { return transport.ValidityDuration; }
+        //    set { transport.ValidityDuration = value; }
+        //}
+
+        [DefaultValue(MsmqDefaults.CustomDeadLetterQueue)]
+        public Uri CustomDeadLetterQueue
+        {
+            get { return _transport.CustomDeadLetterQueue; }
+            set { _transport.CustomDeadLetterQueue = value; }
+        }
+
+        [DefaultValue(MsmqDefaults.DeadLetterQueue)]
+        public DeadLetterQueue DeadLetterQueue
+        {
+            get { return _transport.DeadLetterQueue; }
+            set { _transport.DeadLetterQueue = value; }
+        }
+
+        [DefaultValue(MsmqDefaults.Durable)]
+        public bool Durable
+        {
+            get { return _transport.Durable; }
+            set { _transport.Durable = value; }
+        }
+
+        [DefaultValue(MsmqDefaults.ExactlyOnce)]
+        public bool ExactlyOnce
+        {
+            get { return _transport.ExactlyOnce; }
+            set { _transport.ExactlyOnce = value; }
+        }
+
+        //[DefaultValue(TransportDefaults.MaxReceivedMessageSize)]
+        public long MaxReceivedMessageSize
```

Comment:
Can you add the relevant constant so this attribute can be uncommented? This is used when working with Configuration.